# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cmake/hotfix-cross.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cmake-4.2.1/Source/cmExportCMakeConfigGenerator.cxx.vanilla	2025-12-13 17:46:59.706371297 +0100
+++ cmake-4.2.1/Source/cmExportCMakeConfigGenerator.cxx	2025-12-13 17:47:01.104663473 +0100
@@ -532,6 +532,7 @@
 void cmExportCMakeConfigGenerator::GenerateImportedFileCheckLoop(
   std::ostream& os)
 {
+  return;
   // Add code which verifies at cmake time that the file which is being
   // imported actually exists on disk. This should in theory always be theory
   // case, but still when packages are split into normal and development
--- cmake-4.2.1/Source/cmGeneratorTarget_IncludeDirectories.cxx.vanilla	2025-12-13 17:57:04.733354685 +0100
+++ cmake-4.2.1/Source/cmGeneratorTarget_IncludeDirectories.cxx	2025-12-13 17:57:45.052839161 +0100
@@ -154,7 +154,7 @@
     for (std::string& entryInclude : entry.Values) {
       if (fromImported && !cmSystemTools::FileExists(entryInclude)) {
         tgt->GetLocalGenerator()->IssueMessage(
-          MessageType::FATAL_ERROR,
+          MessageType::LOG,
           cmStrCat(
             "Imported target \"", targetName,
             "\" includes non-existent path\n  \"", entryInclude,
@@ -165,7 +165,6 @@
             "successfully.\n"
             "* The installation package was faulty and references files it "
             "does not provide.\n"));
-        return;
       }
 
       if (!cmSystemTools::FileIsFullPath(entryInclude)) {
